Update porting content (#3850)
magento/devdocs: guides/v2.1/contributor-guide/contributing.md

- [Manual porting](#porting-manual) - Manual process requiring a strong understanding of Git

{: .bs-callout .bs-callout-info }

> Creating back-ports and up-ports are recommended and a best practice, but not required to contribute code. Anyone can create a back-port and up-port for an already merged pull request but for the duplicate ports we give a high priority to merged pull request author.

Reviewer: ```suggestion
Creating back-ports and up-ports are recommended and a best practice, but not required to contribute code. Anyone can create a back-port and up-port for a merged pull request, however, original pull request authors receive higher priority if there are duplicate ports.
```

Author: Hello @jeff-matthews 
Content updated
